src/plugins/condition/components/condition-set.scss

```diff
@@ -1,155 +1,90 @@
-.c-cs-edit {
-    padding: 0;
-}
-
-section {
-    padding-bottom: 5px;
-}
-
-.c-cs__ui__header {
-    background-color: #D0D1D3;
-    padding: 0.4em 0.6em;
-    text-transform: uppercase;
-    font-size: 0.8em;
-    font-weight: bold;
-    color: #7C7D80;
-    display: flex;
-    justify-content: stretch;
-    align-items: center;
-}
-
-.c-cs__ui__header-label {
-    display: inline-block;
-    width: 100%;
-}
-
-.c-cs__ui_content {
-    padding: 0.4em;
-}
-
-.c-cs-ui__label {
-    color: #333;
-}
-
-.c-cs__ui_content .help {
-    font-style: italic;
-    padding: 0.4em 0;
-}
-
-.current-output {
-    text-transform: uppercase;
-    font-weight: bold;
-    margin: 0.4em 0.6em;
-}
-
-.condition-output {
-    color: #999;
-}
-
-.condition-description {
-    color: #333;
-}
-
-.checkbox.custom {
+.c-cs {
     display: flex;
-    align-items: center;
-    margin-left: 0.6em;
-}
-.checkbox.custom span {
-    display: inline-block;
-    margin-left: 0.6em;
-}
-
-.t-test-data-config {
-    margin-top: 5px;
-}
-
-.c-c__condition-collection.is-disabled {
-    opacity: 0.5;
-}
-
-.widget-condition form {
-    padding: 0.5em;
-    display: flex;
-    align-items: center;
-    justify-content: stretch;
-}
-
-.widget-condition form label {
-    flex-grow: 1;
-    margin-left: 0;
-}
-
-.widget-condition form input {
-    min-height: 24px;
-}
+    flex-direction: column;
+    height: 100%;
+    overflow: hidden;
+
+    &__content {
+        display: flex;
+        flex-direction: column;
+        flex: 1 1 auto;
+        overflow: hidden;
+
+        > * {
+            flex: 0 0 auto;
+            overflow: hidden;
+            + * {
+                margin-top: $interiorMarginSm;
```

Comment:
This seems like it should be interiorMargin rather than interiorMarginSm. In edit view it results in a 5px margin but in read view only 3px and the conditions seem a little too tightly packed to my eye.